Web/ForgeTrust.AppSurface.Docs/wwwroot/css/app.css

```
@@ -523,6 +613,42 @@ body {
     display: none;
 }
 
+@media (max-width: 79.999rem) {
+    .docs-outline-shell[data-outline-enhanced="true"] {
+        --docs-outline-compact-bleed: 1rem;
+        position: sticky;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)